Epam.TestAutomation/Epam.TestAutomation.BDD/Steps/SearchSteps.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Threading;
+using Epam.TestAutomation.BDD.DataDrivenTestingTests;
+using Epam.TestAutomation.Core.Browser;
+using Epam.TestAutomation.Utilities.Logger;
+using Epam.TestAutomation.Helper;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using TechTalk.SpecFlow;
+using Epam.TestAutomation.BDD.Steps.GeneralSteps;
+using Epam.TestAutomation.BDD.Steps.Pages;
+using Epam.TestAutomation.BDD.Steps.SharedComponents;
+using Epam.TestAutomation.BDD.Steps;
+using Epam.TestAutomation.BDD;
+using Epam.TestAutomation.Utilities;
+using Epam.TestAutomation.Web.PageObjects.Pages;
+
+namespace Epam.TestAutomation.BDD
+{
+    [Binding]
+    public class SearchByKeywordSteps
+    {
+        private string enteredKeyword;
```

Comment:
you should use scenario context instead of such constants. It was implemented on lesson!